examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -442,6 +444,9 @@ int AppTask::StartNFCTag()
     int result = GetQRCode(setupPinCode, QRCode, chip::RendezvousInformationFlags::kBLE);
     VerifyOrExit(!result, ChipLogError(AppServer, "Getting QR code payload failed"));
 
+    // TODO: Remove replacing spaces with _ after problem described in #415 will be fixed.
+    std::replace(QRCode.begin(), QRCode.end(), ' ', '_');
```

Comment:
Does it add a lot of RAM and Flash usage?

Reply:
Nothing, 0 for SRAM and like 30 B for FLASH.